### PR TITLE
Automatic release

### PR DESCRIPTION
**Pull request by rf-halfspace**

### CODE REVIEW

File: .github/actions/automatic-release/blabla.json
Comment (kerbou) [inline .github/actions/automatic-release/blabla.json:1]: Should this be part of the PR ?

File: .github/actions/automatic-release/Create-GithubReleaseTests.ps1 [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    It "Deletes <expected> when provided <TagName>" {
+        $previousRelease = Invoke-GithubReleaseList -TagName $TagName
+        $previousRelease | Invoke-GithubReleaseDelete
+        if ($Expected) {
+            Should -Invoke -CommandName "gh" -Exactly -Times 2
+        }
+        else {
+            Should -Invoke -CommandName "gh" -Exactly -Times 1
+        }
+    }
```
Comment (djorgensendk) [inline .github/actions/automatic-release/Create-GithubReleaseTests.ps1:52]: Should this be two tests instead, as two different outcomes seems to be acceptable?

File: .github/actions/automatic-release/Create-GitHubRelease.ps1
Comment (kerbou) [inline .github/actions/automatic-release/Create-GitHubRelease.ps1:1]: Rename suggestion: automatic-release --> github-create-release  

Then it fits in nicely with the existing naming of actions which for the majority seems to be {technology}-{verb}-{the_rest}

billede

Comment (rf-halfspace) [reply .github/actions/automatic-release/Create-GitHubRelease.ps1:1]: renamed to github-create-release.

**Review by kerbou — APPROVED** [review]

**Pull request merged**